src/format/mod.rs

```diff
@@ -268,6 +297,8 @@ pub enum Item<'a> {
     Numeric(Numeric, Pad),
     /// Fixed-format item.
     Fixed(Fixed),
+    /// Extra fixed-format item.
+    FixedExt(FixedExt),
```

Comment:
could you remove this from `Item`

Reply:
Agreed. Adding `FixedExt` would be a breaking change. We (now) have `FixedInternal` to but new variants in.